docs/HelpStyleGuide.md

```diff
@@ -66,3 +66,6 @@ This means using more direct language and tone to get information across to the
 * Break up large blocks of text. It's more visually pleasing and less intimidating.
 
 For more information, including research studies conducted, on the use and value of Plain Language, please see [this article](https://www.nngroup.com/articles/plain-language-experts/) and feel free to browse the rest of this website for further helpful details, tips, and tricks. 
+
+In addition to the above guidelines, we use [Alex JS](https://github.com/get-alex/alex)$
+to help us create documentation that is friendly and accessible.
```

Comment:
I don't think this advertisement for a third party tool belongs here.

Reply:
@christinerogers asked that it be mentioned somewhere in the documentation. It felt most relevant to include it in a document outlining our documentation style.

I don't think it's an advertisement so much as just a fact about the project, similar to saying "we use phan to make sure our code doesn't suck"

Reply:
"and if you want to know why we might change particualr language you use, here's the info... "

Reply:
I feel that providing the link to the tool implies indicates that that's the place to find info.